### PR TITLE
mgmt, mysql, add live tests

### PR DESCRIPTION
**Pull request by v-hongli1**
relate to https://github.com/Azure/azure-sdk-for-java/issues/35316

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/mysql/azure-resourcemanager-mysql/src/test/java/com/azure/resourcemanager/mysql/MySqlManagerTests.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            String adminName = "sqlsa" + randomPadding;
+            // @embedmeStart
+            server = mysqlManager.servers()
+                .define(serverName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withProperties(
+                    new ServerPropertiesForDefaultCreate()
+                        .withAdministratorLogin(adminName)
+                        .withAdministratorLoginPassword("!QA2ws#ED4rf")
```
Comment (weidongxu-microsoft) [inline sdk/mysql/azure-resourcemanager-mysql/src/test/java/com/azure/resourcemanager/mysql/MySqlManagerTests.java:95]: Add a comment that user MUST change the password.

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/mysql/azure-resourcemanager-mysql/src/test/java/com/azure/resourcemanager/mysql/MySqlManagerTests.java (hunk 1 above)
Comment (v-hongli1) [reply sdk/mysql/azure-resourcemanager-mysql/src/test/java/com/azure/resourcemanager/mysql/MySqlManagerTests.java]: The password setting has been changed to randomly generated.

Comment (weidongxu-microsoft) [reply sdk/mysql/azure-resourcemanager-mysql/src/test/java/com/azure/resourcemanager/mysql/MySqlManagerTests.java]: OK. Make sure your local run still pass.

**Pull request merged**